README.md

```diff
@@ -12,7 +12,7 @@ recommendations for version control, documentation, or testing.
 
 [The source for this project is available here][src].
 
-Most of the configuration for a Python project is done in the `setup.py` file,
+Most of the configuration for a Python project is done in the `pyproject.toml` file,
```

Comment:
```suggestion
The metadata for a Python project is defined in the `pyproject.toml` file,
```

Reply:
Wording it as suggested avoids the question of whether build backend configuration should/must be done in `pyproject.toml`.